src/vue/modules/fees/components/fees-table.vue

```diff
@@ -60,29 +58,31 @@
               {{ { value: fee.upperBound, currency: fee.asset } | formatMoney }}
             </td>
           </tr>
+          <tr
+            :key="'no-data'"
```

Comment:
better use `<tbody[v-if="fees.length"> ... <tbody[v-else]>` or `<template[v-if="fees.length"> ... <template[v-else]>` instead of key. to decrease performance impact on Vue